### DISCUSSION TITLE
how to RBAC at the table level

### DISCUSSION THREAD

**Discussion by djouallah** [apache/polaris#128] [Q&A] [answered]
playing with Polaris, and it seems RBAC is applied at the namespace level ? is that correct, can I do the same for table ?

**Comment by TomerHeber** [comment, marked as answer]
You can apply RBAC to catalog, namespace, view and table levels.

See image:

![image](https://github.com/user-attachments/assets/7b6d29e8-5438-4c04-8111-27368514bd56)
